### DISCUSSION TITLE
Enhancement to auto populate volunteers

### DISCUSSION THREAD

**Discussion by AGNMM18** [CareTogether/CareTogetherCMS#495] [Ideas]
In the future, is there a possibility for volunteers to auto populate based on their church?   Specifically Ministry Coordinators - more discussion possibly helpful in the future.  Volunteer assignment without access to editing hosting arrangement in the CMS?

**Comment by LarsKemmann** [comment]
> In the future, is there a possibility for volunteers to auto populate based on their church? Specifically Ministry Coordinators - more discussion possibly helpful in the future.

This will initially be handled by the 'Community' features (coming soon), which will let Ministry Coordinators automatically see the referrals that are being served by volunteers in their church. Auto-populating rules can potentially be added to that at some point if needed.

> Volunteer assignment without access to editing hosting arrangement in the CMS?

I'm not sure what this means. Is the ask to be able to have a user role that can make/edit assignments but not change other things about a hosting arrangement? That is possible already. :)

**Reply by AGNMM18** [reply]
Ministry Coordinators would need to know about volunteers who are serving from their church without being able to change anything about the hosting details.  They would have 'view only' access to a hosting arrangement, they cannot make or edit assignments....so we can assign it to them in the capacity that they serve as MC, without having any ability to make changes to the actual arrangements.
